lib/postgres/postgres-pr/connection.rb

```diff
@@ -65,7 +66,7 @@ def initialize(database, user, password=nil, uri = nil)
     # Check if the password supplied is a Postgres-style md5 hash
     md5_hash_match = password.match(/^md5([a-f0-9]{32})$/)
 
-    @conn << StartupMessage.new(PROTO_VERSION, 'user' => user, 'database' => database).dump
+    write_message(StartupMessage.new(PROTO_VERSION, 'user' => user, 'database' => database))
```

Comment:
I swapped to using a `write_message` wrapper around the existing `@conn << Message.new(..).dump` pattern so that it's easier to verify the messages sent in the rspec tests